mainlyetcetera/hang-in-there-boilerplate
# Add linter file

### What does this PR do?

- adds linter file to the directory

### Is this a feature, a fix, or a refactor?

- feature

### Where should the reviewer start?

- directory base

### How should this be tested?

- observe `.eslintrc` under ls -a
- call with `eslint <path_of_file_to_check>`
